monitoring/mongodb/dashboard.json

```
@@ -265,1465 +341,5690 @@
+            "stacking": {
+              "group": "A",
+              "mode": "none"
+            },
+            "thresholdsStyle": {
+              "mode": "off"
+            }
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          },
+          "unit": "bytes"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 11,
+        "w": 12,
+        "x": 12,
+        "y": 148
       },
-      "type": "graph",
-      "xaxis": {
-        "buckets": null,
-        "mode": "time",
-        "name": null,
-        "show": true,
-        "values": []
+      "id": 1594,
+      "options": {
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "right",
+          "showLegend": true
+        },
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
       },
-      "yaxes": [
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "metalk8s-prometheus"
+          },
+          "editorMode": "code",
+          "expr": "mongodb_mongod_wiredtiger_cache_max_bytes{pod=\"data-db-mongodb-sharded-shard0-data-0\", job=~\"zenko/data-db-mongodb-sharded-shard.*-data\"}",
+          "hide": false,
+          "instant": false,
+          "legendFormat": "max",
+          "range": true,
+          "refId": "B"
+        },
         {
-          "format": "ops",
-          "label": null,
-          "logBase": 1,
-          "max": null,
-          "min": null,
-          "show": true
+          "datasource": {
+            "type": "prometheus",
+            "uid": "metalk8s-prometheus"
+          },
+          "editorMode": "code",
+          "expr": "mongodb_mongod_wiredtiger_cache_bytes{job=~\"zenko/data-db-mongodb-sharded-shard.*-data\"}",
+          "hide": false,
+          "instant": false,
+          "legendFormat": "used ({{ pod }})",
+          "range": true,
+          "refId": "A"
         },
         {
-          "format": "short",
-          "label": null,
-          "logBase": 1,
-          "max": null,
-          "min": null,
-          "show": true
+          "datasource": {
+            "type": "prometheus",
+            "uid": "metalk8s-prometheus"
+          },
+          "editorMode": "code",
+          "expr": "mongodb_ss_wt_cache_tracked_dirty_bytes_in_the_cache{job=~\"zenko/data-db-mongodb-sharded-shard.*-data\"}",
+          "hide": false,
+          "instant": false,
+          "legendFormat": "dirty ({{pod}})",
+          "range": true,
+          "refId": "C"
         }
       ],
-      "yaxis": {
-        "align": false,
-        "alignLevel": null
-      }
+      "title": "WiredTiger Cache",
```

redundant with section title
```suggestion
      "title": "Cache Size",
```